runelite-client/src/main/java/net/runelite/client/plugins/achievementdiary/diaries/KourendDiaryRequirement.java

```diff
@@ -0,0 +1,135 @@
+		//  new QuestRequirement(Quest.THE_ASCENT_OF_ARCEUUS));
+		add("Mine some Volcanic sulphur.",
+			new SkillRequirement(Skill.MINING, 42));
+		add("Enter the Farming Guild.",
+			new SkillRequirement(Skill.FARMING, 45));
+		add("Switch to the Necromancy Spellbook at Tyss.",
+			new FavourRequirement(Favour.ARCEUUS, 60));
+		add("Repair a Piscarilius crane.",
+			new SkillRequirement(Skill.CRAFTING, 30));
+		add("Deliver some intelligence to Captain Ginea.",
+			new FavourRequirement(Favour.SHAYZIEN, 40));
+		add("Catch a Bluegill on Molch Island.",
+			new SkillRequirement(Skill.FISHING, 43),
+			new SkillRequirement(Skill.HUNTER, 35));
+		add("Use the boulder leap in the Arceuus essence mine.",
+			new SkillRequirement(Skill.AGILITY, 49));
+		add("Mine some Volcanic sulphur.",
+			new SkillRequirement(Skill.FIREMAKING, 50));
+		add("Catch a Chinchompa in the Kourend Woodland.",
+			new SkillRequirement(Skill.HUNTER, 53),
+			new QuestRequirement(Quest.EAGLES_PEAK));
+		add("Chop some Mahogany logs north of the Farming Guild.",
+			new SkillRequirement(Skill.WOODCUTTING, 50));
+
+		//HARD
+		add("Enter the Woodcutting Guild.",
+			new SkillRequirement(Skill.WOODCUTTING, 60),
+			new FavourRequirement(Favour.HOSIDIUS, 75));
+		add("Smelt an Adamantite bar in the Forsaken Tower.",
+			new SkillRequirement(Skill.HUNTER, 70));
+		add("Kill a Lizardman Shaman in Molch.",
+			new FavourRequirement(Favour.SHAYZIEN, 100));
+		add("Mine some Lovakite.",
+			new SkillRequirement(Skill.MINING, 65),
+			new FavourRequirement(Favour.LOVAKENGJ, 30));
+		add("Plant some Logavano seeds at the Tithe Farm.",
+			new SkillRequirement(Skill.HUNTER, 74),
+			new FavourRequirement(Favour.HOSIDIUS, 100));
+		add("Teleport to Xeric's Heart using Xeric's Talisman.",
+			new QuestRequirement(Quest.ARCHITECTURAL_ALLIANCE));
+		add("Deliver an artefact to Captain Khaled.",
+			new SkillRequirement(Skill.THIEVING, 49),
+			new FavourRequirement(Favour.PISCARILIUS, 75));
+		add("Kill a Wyrm in the Karuulm Slayer Dungeon.",
+			new SkillRequirement(Skill.SLAYER, 62));
+		add("Cast Monster Examine on a Troll south of Mount Quidamortem",
+			new SkillRequirement(Skill.MAGIC, 66),
+			new QuestRequirement(Quest.DREAM_MENTOR));
+
+		//ELITE
+		add("Craft one or more Blood runes.",
+			new SkillRequirement(Skill.RUNECRAFT, 77),
+			new SkillRequirement(Skill.MINING, 38),
+			new SkillRequirement(Skill.CRAFTING, 38),
+			new FavourRequirement(Favour.ARCEUUS, 100));
+		add("Chop some Redwood logs.",
+			new SkillRequirement(Skill.WOODCUTTING, 90),
+			new FavourRequirement(Favour.HOSIDIUS, 75));
+		add("Catch an Anglerfish and cook it whilst in Great Kourend.",
+			new SkillRequirement(Skill.FISHING, 82),
+			new SkillRequirement(Skill.COOKING, 84),
+			new FavourRequirement(Favour.PISCARILIUS, 100));
+		add("Kill a Hydra in the Karruulm Slayer Dungeon.",
+			new SkillRequirement(Skill.SLAYER, 95));
+		add("Create an Ape Atoll teleport tablet.",
```

Comment:
Requires MM1 & RFD Monkey part, as far as I know.

Reply:
Yeah might do I'll have to check, you might beable to make tablet without idk.

Reply:
Wiki seems to think just monkey madness idk abour rfd part. Good spot